### PR TITLE
Dont reorder subnav in new nav, and current section first if it isn't exist

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
This change should do two things.
* Stops the reordering of the subnav list depending on weather you are in a section or not
* If you are in a section that isn't in the nav, it will be added and put at the front.

cc @guardian/dotcom-platform 

## What is the value of this and can you measure success?
Better UX

## Does this affect other platforms - Amp, Apps, etc?
Nope

## Screenshots
Before on an article in the stage section:

After:

No reordering:

## Tested in CODE?
Nope

### CODE REVIEW

**Comment by stephanfowler** [comment]
Perfect!

File: common/app/common/NewNavigation.scala [hunk 1]
```diff
@@ -313,11 +323,13 @@ object NewNavigation {
...
       } else {
         val section = sectionList.head
-        val parentSections = section.parentSection.getPopularEditionalisedNavLinks(edition).drop(1)
+        val parentSection = section.parentSection.getPopularEditionalisedNavLinks(edition).drop(1)
 
-        val (a, b) = parentSections.partition(_.uniqueSection != section.pageId)
-
-        b ++ a
+        if (parentSection.contains(section.navLink)) {
+          section.parentSection.getPopularEditionalisedNavLinks(edition).drop(1)
```
Comment (SiAdcock) [inline common/app/common/NewNavigation.scala]: Return `parentSection` here?

File: common/app/common/NewNavigation.scala [hunk 2]
```diff
@@ -313,11 +323,13 @@ object NewNavigation {
...
+        } else {
+          Seq(section.navLink) ++ section.parentSection.getPopularEditionalisedNavLinks(edition).drop(1)
```
Comment (SiAdcock) [inline common/app/common/NewNavigation.scala]: Would `Seq(section.navLink) ++ parentSection` work?

**Review by SiAdcock — APPROVED** [review]
Couple of questions, otherwise LGTM 💎 💎 👍

**Pull request merged**